downstream/modules/platform/proc-upgrade-controller-hub-eda-unified-services.adoc

```diff
@@ -22,7 +22,7 @@ DO NOT upgrade {EDAName} and the unified UI ({Gateway}) to the latest version of
 
 . Merge 2.4 inventory data into the 2.5 inventory. 
 +
-The example below shows the inventory file for {ControllerName} and {HubName} for 2.4 and the inventory file for {EDAName} and the unified UI ({Gateway}) for 2.5, respectively, as the starting point, and what the merged inventory looks like. 
+The example below shows the inventory file for {ControllerName} and {HubName} for 2.4 and the inventory file for {EDAName} and the unified UI ({Gateway}) for 2.6, respectively, as the starting point, and what the merged inventory looks like. 
```

Comment:
Same as other instances (ensure consistency with how 2.6 is applied -- either use attribute or hard-coded 2.6).